jina/peapods/pods/__init__.py

```diff
@@ -105,7 +105,7 @@ def _parse_args(self, args: Namespace) -> Dict[str, Optional[Union[List[Namespac
             peas_args['head'] = _copy_to_head_args(args, args.polling.is_push)
             peas_args['tail'] = _copy_to_tail_args(args)
             peas_args['peas'] = _set_peas_args(args, peas_args['head'], peas_args['tail'])
-        elif getattr(args, 'uses_before', None) or getattr(args, 'uses_after', None):
+        else:
```

Comment:
I think you should be able to safely do :+1: 
```python
elif args.uses_before or args.uses_after
```

Reply:
thanks, I'm not touching this code anymore